packages/mesos/build

```diff
@@ -17,8 +17,7 @@ LIBS="-lssl -lcrypto" LDFLAGS="-L/opt/mesosphere/active/openssl/lib" "/pkg/src/m
   --with-libevent=/opt/mesosphere/active/libevent \
   --with-curl=/opt/mesosphere/active/curl \
   --sbindir="$PKG_PATH/bin"
-# TODO(cmaloney): -j8 is almost certainly wrong.
-make -j8
```

Comment:
Sure, would love to open this up for more discussion. Just trying to make it so that we're not wasteful or too aggressive with resources depending on what kind of machine it's on.